rfcs/avatar-component.md

```diff
@@ -40,23 +40,23 @@ export default () => (
 * `name: string` - Required
   Defines an alternative text description of the image.
 * `overrides: {}` - Optional
-  * `Avatar: ?React.ComponentType` component to use for Avatar image styling
+  * `Avatar: ?React.ComponentType` component to use for Avatar image styling.
+  * `Root: ?React.ComponentType` component to use for containing element. displays if image fails to load.
```

Comment:
the above two should be `Avatar?: React.ComponentType | {props: {}, style: {}, component: React.ComponentType}` type

Reply:
we have it in our overrides.js types definition. We may use it if customer is aware of where to look for particular type

![image](https://user-images.githubusercontent.com/1851469/49120819-1272d600-f263-11e8-86bf-d087f77605df.png)

Reply:
@blackswanny can you add this comment to the fixit week RFC audit task? i'd rather not change conventions here when we have a specific portion of work to reconsider RFC decisions

Reply:
added